test/fixtures/GH-7131/a.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+require('sys');  // Builtin should not show up in module.children array.
```

Comment:
I picked a module that's unlikely to have already been included by test/common.js.